### DISCUSSION TITLE
Icon touchups

### DISCUSSION THREAD

**Discussion by PickleJesus123** [godotengine/godot-proposals#8278] [Editor]
I love the Godot robot, but I have heard from a large number of people that the icon looks dated and I can't help but agree. I worry that the icon might be turning many folks off, and that complaints should not be simply dismissed/ignored. To kick off the discussion, here are some possible revisions:
![alternatives](https://github.com/godotengine/godot-proposals/assets/12555805/b2d2d80e-c328-44eb-8322-d7413280b497)

**Comment by Calinou** [comment]
- Related to https://github.com/godotengine/godot-proposals/discussions/7304 and https://www.reddit.com/r/godot/comments/17hv3kq/which_do_you_think_looks_more_professional/.

**Comment by PickleJesus123** [comment]
A good quote from u/josh_the_misanthrope:
`a logo that adheres to contemporary graphic design sensibilities conveys that the software is modern.`

**Comment by sonic2kk** [comment]
I, personally, have never once felt the Godot logo needed to change. It has come up innumerable times over the years, but I don't understand the issues. The logo stands out and is memorable, and I don't care much for the notion that a logo has to be "professional." In a sea of oversimplified logos, Godot stands out and has had a very consistent logo with very minor changes over the last decade or so that I've followed the project.

From the screenshots provided, the main issue is that the icon is smaller than the others at least to my eyes.

We should also keep in mind that on Linux (and potentially Android) icons are not fixed, and distributions may package their own default icon sets. So this becomes even less of an issue. Not to mention, from the discussions I have seen at least, there are so many different opinions on which direction a new logo should take, so once again it would be a recurring discussion that I don't personally understand the value of.

On the flip side, as has been brought up before, adoption of a new logo would be an uphill battle. This could lead to confusion, albeit I will say this is a much weaker reason in opposition to a logo change.

If a project has to change its logo to match the current design trend, the discussion of a logo change will always come up, so I'm of the opinion that a change like this is unnecessary and won't solve anything. Judge software on its capabilities, not it's logo. I see the icon as more of a customisation setting to go along with operating system theme, and don't tie it directly to the software itself, other than to identify it from my application list.

**Reply by PickleJesus123** [reply]
Thank you for adding your perspective, sonic2kk.

As you have said, requests for a logo change have come up an innumerable amount of times. Wouldn't that suggest that there are a significant number of users who are unsatisfied with the current logo and would like to see it improved? I hope we aren't dismissing valid concerns, especially if it is brought up so frequently

I worry there is a bit of confusion. It seems that some mistakenly think that we want to get rid of the friendly Godot robot or simplify the design until it is completely forgettable, but this is not the case. We love the Godot robot, and we especially love Godot and everything it stands for. It's just that there's something "off" compositionally about the current icon. From previous polls, it seems that ~40% of Godot users agree, which is sizeable. An average math-inclined software developer might not notice or care for subpar icons, so it might be worthwhile to get some qualified graphic designers to have a look at it and see what they have to say.

**Reply by sonic2kk** [reply]
I'm certainly not trying to dismiss your concerns, however, what I was trying to get across is that while there are definitely requests for a logo change, everyone has their own opinions on what should change.

There was a discussion (I believe linked as the first comment here) that only gave minor changes to the shape of the logo. Others have suggested far more radical changes, moving away entirely from the robot. Your proposals for icons are neither and that's totally fine, as you have said this is just to get a discussion going.

What I'm trying to get at, is that there is broad opinion. While approximately 40% of users may believe that something should change, how many of those 40% could even agree as a majority on a new logo? And if the logo changes, this could come up again in another few years when people feel the logo should get a refresh. And, if the logo is consistently changing even if there's a consistent design pattern, that's something that would be difficult for things like websites, courses, promotional materials etc to keep up with.

So even if many people want a logo change, getting agreement on what the icon should change to is difficult and there has been no solid consensus from what I have seen. If the logo changed there should still be "backlash", so keeping it the same is probably the path of least resistance here.

Getting a graphic designer to weigh in would certainly be interesting, but I don't think the logo should be changed to simply be less "dated", or as another comment put it, more "professional." It should be changed for more of a functional reason if it is to change at all. That's just my perspective.

**Reply by PickleJesus123** [reply]
I strongly agree that having a qualified graphic designer (or multiple designers) give their thoughts on the current logo would be interesting and worthwhile

**Comment by partymetroid** [comment]
I think a fairer comparison if the original icon was rendered at a higher resolution (via svg) and scaled to match the size of the other icons.

**Reply by PickleJesus123** [reply]
The "original" in the top-left is how Godot 4.1 is naturally displayed on my taskbar. Maybe the logo in the exe could be larger?

**Comment by PickleJesus123** [comment]
Just wanted to add one more data point to the discussion:

Often, before these discussions are terminated/dismissed, at least one user will say "Open Source software has bad logos. Get over it". But that is demonstrably false - please see some logos of popular open-source software with modern icons:
![modern2](https://github.com/godotengine/godot-proposals/assets/12555805/2ac1f73d-39ac-478b-9f63-51f8a1334e01)

**Reply by sonic2kk** [reply]
This kind of logo is exactly the kind I would like Godot to avoid, and is imo a good argument to stay with the current logo. Firefox especially is a logo change I saw significant backlash over and still see it to this day, the trend of unwanted "oversimplified" logos has become a running joke.

And I have seen the argument that FOSS has had design quite a lot so I'm not dismissing that, I absolutely see the point you're making, though in my opinion the updated Firefox logo is pretty bad. I actually use the old logo, overriding the one set by my icon theme.

And finally we should also keep in mind that there are many FOSS icon sets for Linux distributions that may have more broadly appealing icons. So absolutely, open source software need not have a "bad" logo by "tradition." Improvement is always good, though agreement on what is an improvement is difficult.

**Reply by PickleJesus123** [reply]
I appreciate your contribution to the discussion, and I aim to be as unbiased as possible

`Firefox especially is a logo change I saw significant backlash over`
Really? After [a quick Google search](https://www.google.com/search?q=firefox+new+logo+reddit), the new 2019 logo seems to be the most widely liked. Obviously some people were upset for nostalgia reasons, but there were unquestionably the minority.

**Reply by sonic2kk** [reply]
In the search you linked, which is primarily Reddit posts, [there is acknowledgement that the logo change was not well received](https://www.reddit.com/r/pcmasterrace/comments/lr86rq/i_think_the_current_firefox_logo_looks_good/).

I wouldn't say I prefer the older logo for nostalgia reasons, but rather I think it stands out more. I'm not a big fan of these oversimplified logos, things that resemble the likes of the macOS or Windows 11 design language are to be avoided in my opinion.

**Reply by PickleJesus123** [reply]
I wonder if the Google searches differ based on location. [The posts I am seeing](https://www.reddit.com/r/firefox/comments/vaxu47/which_version_of_the_firefox_logo_is_your/) paint a different picture, where "newer"/"fresher" logos are more popular

**Reply by sonic2kk** [reply]
That is possible, though [the first comment](https://www.reddit.com/r/firefox/comments/vaxu47/comment/ic5kvbq/) on the post you linked explains in large part why the new logo is disliked. Other comments share a similar sentiment.

And while the majority in that poll picked the newer logo, response rate was low, and post has 0 votes. So it seems it was not a very popular poll, though that is just one example.

If anything though, I believe this outlines the split in opinion if nothing else on logo design.

And, for the record, 2013 Firefox is my favourite, though I recognise that *that* opinion is unpopular. I prefer that design language.

**Reply by PickleJesus123** [reply]
Those are fair points.

The conclusion I am coming to is that there is a lot of uncertainty and polarization around the logo. Given how frequently the logo issue is brought up, I don't think it would hurt to show the Godot logo to a qualified graphic designer(s) and just see what they have to say.

I just want Godot to be the best it can be

**Reply by sonic2kk** [reply]
I would agree, though at the same time, I still don't understand the importance personally. If a logo is going to put someone off using software, are they serious about using it to begin with?

I may dislike the new Firefox logo, quite a lot even, but it didn't even factor into my decision to switch. The speed, low resource usage, extensions, and above all first-class Wayland support, all played a much greater role. In other words, the feature and functionality.

I would be more interested in seeing just how serious the logo issue is and how much the logo *actually* plays into a decision like this. Especially because a logo is a very easy thing to change. I'm not saying it doesn't because *I don't know*, and that's what I would be interested in knowing. It may even make this of greater importance and may be the more valuable data point.

**Reply by PickleJesus123** [reply]
I am only bringing up this whole logo thing because I saw [this popular video (timestamp 3:47)](https://youtu.be/zHSr9VETfdE?si=rvu8ejj5OuYqseE3&t=227) when I was deciding whether I should switch from Unity to Godot. In the end I switched fully to Godot, but things like the logo made me hesitant at first. I couldn't help but think "if they aren't willing to keep something as essential and forward-facing as a logo up to date with current standards, what else is neglected and outdated behind the scenes?". My lizard brain was using the logo as a heuristic to gauge the health/modernity of the entire project, and I don't think that was an unreasonable thing to do

**Reply by sonic2kk** [reply]
I watched the clip of that video, and I disagree with the points made. I'm not convinced a logo sends that much of a message, but again that is just me personally. In my world, a logo is something set by the operating system first and foremost, and plays into the look-and-feel of the OS rather than anything specific to the software itself. I put much more stock in the UIX of the engine than the logo, which I barely even acknowledge. There are plenty of other things that have much more value to me with any software than the logo.

I would much sooner decide whether to use software based on UIX and functionality than the logo it uses on my application launcher. A logo is something I'll acknowledge in passing, but as long as it's there and isn't intrusive, I don't think a logo needs to change all that much.

**Comment by sonic2kk** [comment]
As the subject of getting a graphic designer to evaluate the current Godot logo has come up before, I would like to ask: Have the motivations behind the current design already been discussed? I believe there were some minor logo adjustments a number of years back but to my understanding the general logo has stayed the same. Were the motivations behind *that* touch-up discussed anywhere in these kinds of discussions? The logo change discussion is not new, so it's very likely that this has been discussed to death.

This isn't accusatory, I'm just not sure how else to phrase it. I'm just wondering if we could get background on where the current logo came from before any serious stock is put into getting someone to evaluate its change. Understanding how the current logo was designed could help strengthen the position of keeping the current logo, moreso than the position individuals like me take, where I'm not a designer, I just don't see the need for the logo to change and can list various non-designer related reasons.

I like the current logo, I think it's fine, and if the logo changes I can always use a different one or use a different icon theme to change it. But since it's been brought up about getting a designer in to look at the logo, perhaps some background on how the current general logo design came to be would be useful.
